src/Tooltip/__tests__/Tooltip.test.js

```diff
@@ -16,7 +16,20 @@ describe('<Tooltip />', () => {
     expect(container.firstChild).toMatchSnapshot();
   });
 
-  test('should render open a tooltip withouth a problem', () => {
+  test('should render without a problem with white appearance', () => {
```

Comment:
don't use a snapshot here test properties for the light appearance with [jest styled component](https://github.com/styled-components/jest-styled-components)